cki_lib/libcki.sh

```diff
@@ -8,7 +8,30 @@
 #       2) All global variables start with 'CKI_'.
 #
 
-source /usr/bin/rhts_environment.sh
+# Set CKI test environment
+if [ -z "$OUTPUTFILE" ]; then
+        export OUTPUTFILE=`mktemp /mnt/testarea/tmp.XXXXXX`
+fi
+
+if [ -z "$ARCH" ]; then
+        ARCH=$(uname -i)
+fi
+
+if [ -z "$FAMILY" ]; then
+        FAMILY=$(cat /etc/redhat-release | sed -e 's/\(.*\)release\s\([0-9]*\).*/\1\2/; s/\s//g')
+fi
+
+# Set well-known logname so users can easily find
+# current tasks log file.  This well-known file is also
+# used by the local watchdog to upload the log
+# of the current task.
+if [ -h /mnt/testarea/current.log ]; then
+        ln -sf $OUTPUTFILE /mnt/testarea/current.log
```

Comment:
the background is:
since rhts environment  script for initial setting test environment, is deprecated in restraint. 
rhts-enviromnet got copied to lib_cki without modification

is identical with rhts-envirnment.sh from restraint-rhts.rpm (package)